### PR TITLE
Unbreak build with Clang 9.0.0

### PR DESCRIPTION
**Pull request by jbeich**
See [error log](https://github.com/intel/compute-runtime/files/3933739/compute-runtime-19.48.14977.log)

### CODE REVIEW

File: core/helpers/preamble.h [hunk 1]
```diff
@@ -82,9 +82,9 @@ struct L3CNTLRegisterOffset {
 
 namespace DebugModeRegisterOffset {
 template <typename GfxFamily>
-constexpr uint32_t registerOffset = 0x20ec;
+static constexpr uint32_t registerOffset = 0x20ec;
```
Comment (jbeich) [inline core/helpers/preamble.h]: Only reg_configs.h change is necessary to unbreak build. This one is for consistency.

**Comment by JacekDanecki** [comment]
As I can see your patch works only with clang, all builds using gcc failed.
I've reproduced this issue under Arch with clang-9.

**Comment by JacekDanecki** [comment]
Fix for compilation with clang 9 was committed to [test repository](https://github.com/JacekDanecki/neo-semaphore/commit/c5fa1f7e52844f150aa77de542888dd1a0a228de).
Once it passes all tests it'll be merged and I'll close this PR.

@jbeich Can you verify whether above commit fixes compilation issue under FreeBSD?

**Comment by jbeich** [comment]
> Can you verify whether above commit fixes compilation issue under FreeBSD?

I confirm, [builds fine](https://github.com/intel/compute-runtime/files/3952149/compute-runtime-19.48.14977.log).

**Comment by JacekDanecki** [comment]
Closing as new pull request #241 was provided.

**Pull request closed without merging**